2-mandatory/4-people-i-know.js

```diff
@@ -393,7 +403,13 @@ You can see who people's colleagues are by seeing the "colleagues" array in each
 This time, I only want the full names ("<firstname> <lastname>") of my friends who are colleagues of hers.
 */
 
-let friendsWhoAreColleaguesOfStacie = [];
+let friendsWhoAreColleaguesOfStacie = friends.filter((friend) =>
+  friend.colleagues.some((colleague) => colleague.name === "Stacie Villarreal")
+);
+friendsWhoAreColleaguesOfStacie = friendsWhoAreColleaguesOfStacie.map(
+  (friend) => `${friend.name.first} ${friend.name.last}`
```

Comment:
```suggestion
  ({ name }) => `${name.first} ${name.last}`
```
Functionally everything is correct, but we can use destructuring of parameters to make the code easier to read if we have a lot of nesting levels. :)